### DISCUSSION TITLE
Github学生申请似乎无法检测以中文书写的证明材料/The Github student application seems to be unable to test the supporting materials written in Chinese.

### DISCUSSION THREAD

**Discussion by cyrxdzj** [community/community#53486] [GitHub Education]
在申请Github学生优惠中，Github要求提供学生ID卡等证明材料。但是，我一旦提交我的证明材料，系统便立刻提示：
In applying for Github student discount, Github is required to provide proof materials such as student ID card. However, as soon as I submit my proof, the system immediately prompts:
![1682170153578](https://user-images.githubusercontent.com/63708985/233787969-52e063e4-fd56-4efb-835f-aadc4eb3a40b.png)
请问这个问题如何解决？谢谢！
May I ask how to solve this problem? Thank you!
The English part is translated by Tencent Mr. Translator.

**Comment by samus-aran** [comment]
Hi @cyrxdzj 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Reply by cyrxdzj** [reply]
谢谢！不过学信网的学历报告只有上大学才可用吧，我初三。。。不过也没事，我耐心等到大学即可。
